### PR TITLE
National power rangers' day data and images!

### PR DESCRIPTION
**Pull request by TinasheMTapera**
Thank you for your submission! Please leave this text as-is when you submit, and then check the checklist.

Usability:

- [x] This dataset has not already been used in TidyTuesday.
- [x] The dataset is less than 20MB when saved as a tidy CSV.
- [x] I can imagine at least one data visualization related to this dataset.

Preparation:

- [x] `cleaning.R` has code to download and clean the dataset, resulting in descriptively-named data.frames.
- [x] I ran `saving.R` to create `csv`s and `md` data dictionaries for each of my data.frames.
- [x] I filled in descriptions for each variable in each of the `{dataset}.md` files.
- [x] I edited the `intro.md` file to introduce my dataset.
- [x] I included at least one image for my dataset as a `png` file.
- [x] I completed the information in `meta.yaml`, including descriptive alt text for each image.
- [x] I provided information in `meta.yaml` about how to credit me, and deleted any parts of that block that I do not want you to use.

### CODE REVIEW

File: data/curated/power_rangers/intro.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!-- 
+1. Describe the dataset. See previous weeks for the general format of the
+DESCRIPTION. The description is the part of the readme.md file above "The Data";
+everything else will be filled in from the other md files in this directory +
+automatic scripts. We usually include brief introduction along the lines of
+"This week we're exploring DATASET" or "The dataset this week comes from 
+SOURCE", then a quote starting with ">", then a few questions participants might
+seek to answer using the data. 
+2. Delete this comment block.
+-->
```
Comment (jonthegeek) [inline data/curated/power_rangers/intro.md:10]:
```suggestion
```

File: data/curated/power_rangers/intro.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+DESCRIPTION
+
```
Comment (jonthegeek) [inline data/curated/power_rangers/intro.md:12]:
```suggestion
```

File: data/curated/power_rangers/intro.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+> In 1993, five ordinary teenagers exploded on the pop-culture scene with the launch of Mighty Morphin Power Rangers. Together they broke down barriers. They defeated evil by demonstrating teamwork, inclusivity, and diversity to people of all ages. Today, this grand tradition continues as new Ranger teams and new generations of fans discover these essential values again.
+The series, created by Haim Saban, has one of the most popular taglines in history, "It's Morphin Time!" The TV series "Mighty Morphin Power Rangers" (MMPR) launched on August 28, 1993. Power Rangers quickly became the #1 kids action brand and a global phenomenon. With its current 25th season, "Power Rangers Super Ninja Steel," the show is now the second longest-running non-soap opera scripted program on American TV after "The Simpsons." There are also over 830+ episodes in its library. Currently, Power Rangers is seen in more than 150 markets around the world. It's also translated into numerous languages and a favorite on many indispensable children's programming blocks around the world. Go Go Power Rangers on 8.28!
```
Comment (jonthegeek) [inline data/curated/power_rangers/intro.md:16]:
```suggestion
The series, created by Haim Saban, has one of the most popular taglines in history, "It's Morphin Time!" The TV series "Mighty Morphin Power Rangers" (MMPR) launched on August 28, 1993. Power Rangers quickly became the #1 kids action brand and a global phenomenon. With its current 25th season, "Power Rangers Super Ninja Steel," the show is now the second-longest-running, non-soap-opera, scripted program on American TV (after "The Simpsons"). There are also over 830 episodes in its library. Currently, Power Rangers is seen in more than 150 markets around the world. It's also translated into numerous languages and is a favorite on many indispensable children's programming blocks around the world. Go Go Power Rangers on 8.28!
```

File: data/curated/power_rangers/intro.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Source: [NationalDayCalendar.com](https://www.nationaldaycalendar.com/national-day/national-power-rangers-day-august-28)
+
+QUESTION?
+
```
Comment (jonthegeek) [inline data/curated/power_rangers/intro.md:21]:
```suggestion
QUESTION?

```

File: data/curated/power_rangers/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+title: "This week we're exploring The Power Rangers Franchise!"
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:1]:
```suggestion
title: The Power Rangers Franchise
```

File: data/curated/power_rangers/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+article:
+  title: NATIONAL POWER RANGERS DAY | August 28
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:3]:
```suggestion
  title: National Power Rangers Day (August 28)
```

File: data/curated/power_rangers/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  url: https://www.nationaldaycalendar.com/national-day/national-power-rangers-day-august-28
+data_source:
+  title: "Power Rangers: Seasons and episodes data"
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:6]:
```suggestion
  title: Power Rangers: Seasons and episodes data
```

File: data/curated/power_rangers/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  url: https://www.kaggle.com/datasets/karetnikovn/power-rangers-dataset/data
+images:
+# Please include at least one image, and up to three images
+- file: National-Power-Rangers-Day.png
+  alt: National Power Rangers Day Mighty Morphin' Power Rangers
+- file: national-power-rangers-day--august-28.png
+  alt: National Power Rangers Day Image by NationalDayCalendar.com
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:13]:
```suggestion
  alt: National Power Rangers Day, August 28, by NationalDayCalendar.com #CelebrateEveryDay. cartoon representations of the Power Rangers face off against some of their iconic villains.
```

File: data/curated/power_rangers/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+credit:
+# If you do not want a particular type of credit, please delete the related line
+  post: Nikita Karetnikov
+  linkedin: https://www.linkedin.com/in/nikita-karetnikov-683095138/
+  kaggle: https://www.kaggle.com/karetnikovn
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:18]: The credit is for *you*, not the person who the data comes from. Right now it only shows up in the dataset's readme, but the intention is to eventually also use these credits in social media posts.

File: data/curated/power_rangers/power_rangers_episodes.md
Comment (jonthegeek) [inline data/curated/power_rangers/power_rangers_episodes.md:1]: Please replace the prefilled text in the "description" column with a brief description of the content of the column.

File: data/curated/power_rangers/power_rangers_seasons.md
Comment (jonthegeek) [inline data/curated/power_rangers/power_rangers_seasons.md:1]: Please replace the prefilled text in the "description" column with a brief description of the content of the column.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
I thought I could make a few tweaks and be done, but there are a couple things that could use your input. Thank you for knocking this out! Note the things I changed, so you're ready for next time, but the most important things are to fill in your credit & the field descriptions in the dictionary!

File: data/curated/power_rangers/meta.yaml (hunk 9 above)
Comment (TinasheMTapera) [reply data/curated/power_rangers/meta.yaml:18]: Gotcha, added my info here

**Comment by TinasheMTapera** [comment]
Hi @jonthegeek, I've amended the changes in  6a14079

File: data/curated/power_rangers/power_rangers_episodes.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+|variable      |class     |description   |
+|:-------------|:---------|:-------------|
+|season_title  |character |season_title  |
+|episode_num   |double    |episode_number   |
+|episode_title |character |episode_title |
+|air_date      |double    |air_date      |
+|IMDB_rating   |double    |IMDB_rating   |
+|total_votes   |double    |total_votes   |
+|desc          |character |free_text_description_of_episode|
```
Comment (jonthegeek) [inline data/curated/power_rangers/power_rangers_episodes.md:9]:
```suggestion
|season_title  |character |title of the overall season  |
|episode_num   |double    |number of this episode within this season |
|episode_title |character |title of this episode |
|air_date      |double    |date on which this episode first aired in the U.S. |
|IMDB_rating   |double    |average rating among IMDB users |
|total_votes   |double    |total votes on IMDB |
|desc          |character |free-text description of this episode |
```

File: data/curated/power_rangers/power_rangers_seasons.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+|variable           |class     |description        |
+|:------------------|:---------|:------------------|
+|season_title       |character |season_title       |
+|season_num         |double    |season_number         |
+|number_of_episodes |double    |number_of_episodes |
+|air_date_first_ep  |double    |air_date_first_episode  |
+|air_date_last_ep   |character |air_date_last_episode   |
+|producer           |character |producer           |
+|IMDB_rating        |double    |IMDB_rating        |
```
Comment (jonthegeek) [inline data/curated/power_rangers/power_rangers_seasons.md:9]:
```suggestion
|season_title       |character |title of this season |
|season_num         |double    |season number |
|number_of_episodes |double    |number of episodes in this season |
|air_date_first_ep  |double    |date on which the first episode in this season first aired in the U.S. |
|air_date_last_ep   |character |date on which the last episode in this season first aired in the U.S. |
|producer           |character |the company that produced this season |
|IMDB_rating        |double    |average rating of this seasons among IMDB users |
```

**Review by jonthegeek — COMMENTED** [review]
Thank you! I was already planning to update the pre-filled text for the descriptions in the data dictionary, and you helped me see that the snake-case text just makes things confusing. I'll fix that to make it easier for the next contributor 😊

Thanks again!

File: data/curated/power_rangers/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,18 @@
+title: The Power Rangers Franchise
+article:
+  title: National Power Rangers Day (August 28)
+  url: https://www.nationaldaycalendar.com/national-day/national-power-rangers-day-august-28
+data_source:
+  title: Power Rangers: Seasons and episodes data
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:6]:
```suggestion
  title: |
    Power Rangers: Seasons and episodes data
```

File: data/curated/power_rangers/meta.yaml [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+  url: https://www.kaggle.com/datasets/karetnikovn/power-rangers-dataset/data
+images:
+# Please include at least one image, and up to three images
+- file: National-Power-Rangers-Day.png
+  alt: National Power Rangers Day Mighty Morphin' Power Rangers
+- file: national-power-rangers-day--august-28.png
+  alt: National Power Rangers Day, August 28, by NationalDayCalendar.com #CelebrateEveryDay. cartoon representations of the Power Rangers face off against some of their iconic villains.
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:13]:
```suggestion
  alt: National Power Rangers Day, August 28, by NationalDayCalendar.com. cartoon representations of the Power Rangers face off against some of their iconic villains.
```

File: data/curated/power_rangers/meta.yaml (hunk 12 above)
Comment (jonthegeek) [reply data/curated/power_rangers/meta.yaml:6]: Oops, can't have colons in YAML without special treatment!

File: data/curated/power_rangers/meta.yaml [hunk 14]
```diff
@@ -0,0 +1,19 @@
+title: The Power Rangers Franchise
+article:
+  title: National Power Rangers Day (August 28)
+  url: https://www.nationaldaycalendar.com/national-day/national-power-rangers-day-august-28
+data_source:
+  title: |
+    Power Rangers: Seasons and episodes data
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:7]:
```suggestion
  title: "Power Rangers: Seasons and episodes data"
```

File: data/curated/power_rangers/meta.yaml [hunk 15]
```diff
@@ -0,0 +1,18 @@
+title: The Power Rangers Franchise
+article:
+  title: National Power Rangers Day (August 28)
+  url: https://www.nationaldaycalendar.com/national-day/national-power-rangers-day-august-28
+data_source:
+  title: "Power Rangers: Seasons and episodes data"
+  url: https://www.kaggle.com/datasets/karetnikovn/power-rangers-dataset/data
```
Comment (jonthegeek) [inline data/curated/power_rangers/meta.yaml:7]:
```suggestion
  url: "https://www.kaggle.com/datasets/karetnikovn/power-rangers-dataset/data"
```

**Review by jonthegeek — APPROVED** [review]
All set! Thanks again!

**Pull request merged**